Description: Fix for cmake lua support to work around problem that lua needs multiple include directories on Debian
Author: Andrew Ross <andrewross@users.sourceforge.net>
Last-Update: 2012-07-19

===================================================================
Index: plplot-5.9.10/cmake/modules/lua.cmake
===================================================================
--- plplot-5.9.10.orig/cmake/modules/lua.cmake	2013-10-01 13:10:33.183293316 +0100
+++ plplot-5.9.10/cmake/modules/lua.cmake	2013-10-01 13:10:33.171293317 +0100
@@ -86,3 +86,11 @@
     set(LUA_DIR ${LIB_DIR}/lua/${LUA_VERSION}/plplot)
   endif(LUA_VERSION MATCHES "5\\.0")
 endif(ENABLE_lua)
+
+if(ENABLE_lua AND LUA51_FOUND)
+  pkg_check_pkgconfig(lua5.1 includedir libdir linkflags1 cflags1 version1 _LUA)
+  if(linkflags1)
+    set(LUA_INCLUDE_DIR ${includedir})
+  endif(linkflags1)
+endif(ENABLE_lua AND LUA51_FOUND)
+
